crates/pop-cli/src/commands/bench/overhead.rs

```diff
@@ -230,7 +228,10 @@ mod tests {
 	use super::*;
 	use crate::{
 		cli::MockCli,
-		common::bench::{get_mock_runtime, EXECUTED_COMMAND_COMMENT},
+		common::{
+			bench::EXECUTED_COMMAND_COMMENT,
+			runtime::{get_mock_runtime, RuntimeFeature},
```

Comment:
Since it is used multiple times this would be a nice way to reduce the amount of code:
```suggestion
			runtime::{get_mock_runtime, RuntimeFeature::Benchmark},
```

Could you do this in the other files as well if you agree :)

Reply:
Agree. Resolved in https://github.com/r0gue-io/pop-cli/pull/490/commits/dfea031514943830b1bce7a560c6fd96aff22f31